compiler/runtime/OMRCodeCacheManager.hpp

```diff
@@ -144,8 +144,7 @@ class OMR_EXTENSIBLE CodeCacheManager
    void  freeMemory(void *memoryToFree);
 
    TR::CodeCache * allocateCodeCacheObject(TR::CodeCacheMemorySegment *codeCacheSegment,
-                                           size_t codeCacheSizeAllocated,
-                                           CodeCacheHashEntrySlab *hashEntrySlab);
+                                           size_t codeCacheSizeAllocated);
```

Comment:
This parameter sounds incorrect. Shouldn't it be `codeCacheSize` or `codeCacheSizeToAllocate`? "Allocated" implies a return value and this parameter is not of a reference type. Might be a good chance to improve this.

Reply:
Yes, that's a good point.  I think it was named as such because at its call site its argument is named `codeCacheSizeAllocated` which is a reference argument to `getNewCacheMemorySegment`.  However, there's no need to carry that naming convention to the parameter of this API.  I'll change it.